kolibri_explore_plugin/assets/src/views/SearchPage.vue

```diff
@@ -164,41 +166,58 @@
         return this.query.trim();
       },
       resultChannels() {
-        const { channels } = this.searchResult;
-        if (!channels || !channels.length) {
-          return null;
-        }
+        let allChannels = kinds.map((k) => {
+          if (!this.searchResult[k]) {
+            return [];
+          }
 
-        return _.chunk(this.searchResult.channels, this.columns);
+          const { channels } = this.searchResult[k];
+          return channels || [];
+        });
+
+        allChannels = _.flatten(allChannels);
```

Comment:
If you decide to go with map/reduce:
```
[[0, 1, 2], [3, 4, 5]].reduce(
  function(a, b) {
    return a.concat(b);
  },
  []
);
```
Or otherwise `_.flatten(map(...)) `

But please stay functional and avoid `let` variables.